From c365b97c97f902a201407ba5015a6f999bdaf4ba Mon Sep 17 00:00:00 2001
From: Srinivas Neeli <srinivas.neeli@xilinx.com>
Date: Fri, 31 Jul 2020 14:03:39 +0530
Subject: [PATCH 1496/1851] gpio: gpio-xilinx: Initialize ngpio variable in the
 declaration

commit ee5c57a6ffdc3845a28a89821f581cfb7675d9b0 from
https://github.com/Xilinx/linux-xlnx.git

Initialize ngpio variable with zero in the declaration.

Addresses-Coverity:"UNINIT(Using uninitialized value)"
Signed-off-by: Srinivas Neeli <srinivas.neeli@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/gpio/gpio-xilinx.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpio/gpio-xilinx.c b/drivers/gpio/gpio-xilinx.c
index 4d5adc255866..3467cc86fd7f 100644
--- a/drivers/gpio/gpio-xilinx.c
+++ b/drivers/gpio/gpio-xilinx.c
@@ -601,7 +601,7 @@ static int xgpio_of_probe(struct platform_device *pdev)
 	struct xgpio_instance *chip, *chip_dual;
 	int status = 0;
 	const u32 *tree_info;
-	u32 ngpio;
+	u32 ngpio = 0;
 	u32 cells = 2;
 
 	chip = devm_kzalloc(&pdev->dev, sizeof(*chip), GFP_KERNEL);
-- 
2.31.1

